---
title: "Mutation rate"
output:
  html_document:
    css: style.css
author: J. Grey Monroe
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

The principles behind common methods to identify genes contributing to adaptation and phenotypic variation are closely related to the mutation rate of adaptive alleles. And mutation is rate is closely related to the functional molecular basis of adaptation. Specifically, a low mutation rate is expected if only those which improve gene function are adaptive, but can be high if loss-of-function is adaptive. This is simply because the number of mutations that can break a gene is likely large. Indeed support for a low mutation rate has pointed to examples from mutation accumulation in single essential genes rather than genome wide (Jensen 2014) , an error dating back to the early days when genes like hemoglobin inspired theory.  Results also show that the distribution of fitness effects depends on environment. Previous reports of distribution need to be taken with grain of salt because we don’t know how adapted the population was to the environment. In the case of local adaptation, we would expect population to be poorly adapted when expanding into new habitats or when environments change.  The other key consideration is that the distribution differs from gene to gene. While as a whole, the mutation rate of beneficial alleles may be low, for some genes it may be high. Previous results used to support functionalist paradigm fail to tell us whether the low number of beneficial mutations are distributed across few or many genes. 
We recognize that most loss-of-funciton variants are likely deleterious. But what is not made clear from current data is whether the rare beneficial alleles reflect rare gain of function or rare genes in which loss-of-function is beneficial. Nevertheless, the frequency of beneficial loss-of-function may be greater than once expected (Arabidopsis T-DNA experiments).
Estimating DFE from pop gen data is flawed because it assumes beneficial alleles will sweep to fixation. This is not expected when there are competing alleles. 
Few genes, many genes…<sup>[1](#myfootnote1)</sup>


```{r cars}
summary(cars)
```

## Including Plots

You can also embed plots, for example:

```{r pressure, echo=FALSE}
plot(pressure)
```

Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the plot.

<a name="myfootnote1">1</a>: Footnote content goes here
